cmake/external_png.cmake

```diff
@@ -0,0 +1,131 @@
+include(ExternalProject)
```

Comment:
I see your point and I will make the changes. My arguments was that I never considered cmake files as scripts :). It has also been established in many open source projects to put cmake modules in a folder called `cmake/modules/*.cmake` or `cmake/*.cmake`. It might be due to the fact that `CMake` itselfs ships all its modules into `<cmake_install_path>/share/cmake/Modules`.

Reply:
That's okay then. It's not a mandatory change after all :)